pom.xml

```diff
@@ -53,28 +54,28 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>${maven-surefire-plugin.version}</version>
                     <configuration>
-                        <systemPropertyVariables>
-                            <chutney.environment.rootPath>server_conf/conf/environment</chutney.environment.rootPath>
-                            <chutney.report.rootPath>target/chutney/reports</chutney.report.rootPath>
-                        </systemPropertyVariables>
+                        <skip>true</skip>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-failsafe-plugin</artifactId>
+                    <version>3.0.0-M7</version>
```

Comment:
In property like surefire. It seems that those two plugins from Surefire follow the same versionning cycle. Maybe one property coud be enough. Version M8 exists.